api/src/Page/Shipment.php

```diff
@@ -925,6 +1000,27 @@ function _dispatch_dewar() {
                 array($dew['DEWARID'], $dewar_location)
             );
 
+            $data = $this->args;
+
+            $terms = $this->db->pq(
+                "SELECT cta.couriertermsacceptedid FROM couriertermsaccepted cta WHERE cta.shippingid=:1",
+                array($dew['SHIPPINGID'])
+            );
+            $terms_accepted = sizeof($terms) ? true : false;
+
+            if (isset($use_shipping_service) && $use_shipping_service && $terms_accepted && in_array($country, $facility_courier_countries)) {
```

Comment:
2 questions:
1. Did we want to query the database if use_shipping_service is null
2. If shipping service url is null did we want to dispatch it using the shipping service?